### PR TITLE
Added test cases to fix issue#106

### PR DESCRIPTION
**Pull request by pooja1pathak**
## Proposed changes

Added test cases for issue #106. This PR is an add-on fix on PR #498 so I have not added the changes which are already in that PR.

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [CLA][cla]
- [x] I have added tests that prove my fix is effective or that my
      feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion
by explaining why you chose the solution you did and what alternatives
you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/ngsi-timeseries-api/master/individual-cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/smartsdk/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"

### CODE REVIEW

**Comment by pooja1pathak** [comment]
@chicco785 please review this PR and let me know in case any change is required.

**Comment by chicco785** [comment]
> @chicco785 please review this PR and let me know in case any change is required.

sure, the test are failing, I suppose this is due to the fact that the other PR is not yet in the repo. correct?

**Comment by pooja1pathak** [comment]
Yes you are correct, this PR is dependent on PR #498 after merging tests should pass.

**Comment by chicco785** [comment]
@c0c0n3 made a major refactor of the tests to wait for data consolidation in crate. can you rebase your pr?

**Comment by chicco785** [comment]
there are some errors in the tests

**Comment by pooja1pathak** [comment]
> there are some errors in the tests

yes, I will update the tests today

**Comment by pooja1pathak** [comment]
@chicco785 I have updated the tests and it is passing now. I am not sure about autopep8 test which is failing. Can you please look into it and let me know if any change is required from my side.

**Comment by c0c0n3** [comment]
@pooja1pathak so our problem is that we can't run autopep on PRs from an external repos. The only way forward would be that:

1. you run autopep locally---just run the `lint.py.sh` script in the repo root
2. check the tool reformatted the code---git diff
3. if there are no errors, push your branch again

Thanks soooo much!

**Comment by pooja1pathak** [comment]
@c0c0n3 I have updated the code as per your suggestion.

File: timescale-container/quantumleap-db-setup.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ class CreateDb:
     OWNER ${db_user}
     ENCODING 'UTF8';
 
-\connect ${db_name}
+\\connect ${db_name}
```
Comment (chicco785) [inline timescale-container/quantumleap-db-setup.py:208]: why this change? this may break the db setup, correct @c0c0n3 ?

Comment (c0c0n3) [reply timescale-container/quantumleap-db-setup.py:208]: yep, good catch! @pooja1pathak can you confirm you didn't do this yourself? I think it was autopep. I'm busy opening an issue about it. In the meantime, @pooja1pathak  if you could please undo this change, i.e. the deleted line is correct.

Comment (c0c0n3) [reply timescale-container/quantumleap-db-setup.py:208]: @chicco785 @pooja1pathak see #618

Comment (pooja1pathak) [reply timescale-container/quantumleap-db-setup.py]: > yep, good catch! @pooja1pathak can you confirm you didn't do this yourself? I think it was autopep. I'm busy opening an issue about it. In the meantime, @pooja1pathak if you could please undo this change, i.e. the deleted line is correct.

No, it was autopep. I have reverted the change.

**Comment by pooja1pathak** [comment]
@chicco785 @c0c0n3 I am done with the changes and updated the PR. Please review the PR and let me know if I need to do any changes for failing coverage test.

**Comment by c0c0n3** [comment]
@pooja1pathak thank you so much for your PR!

> let me know if I need to do any changes for failing coverage test.

No I don't think there's anything you can do about it. It looks like the pipeline is badly stuck on Circle CI. It probably failed to upload the coverage report the first time it ran which is why the coverage check failed. I tried triggering the pipeline again, but it doesn't seem to have any effect---i.e. the build doesn't start over. 

@chicco785 can we merge this PR even if the coverage check failed? Or should we try figuring out how to make it pass?

**Comment by chicco785** [comment]
it's coverall failing... not clue what's the reason, but it cannot be @pooja1pathak code
